cron/notification-for-bekrachtigde-mandataris.ts

```diff
@@ -42,14 +38,6 @@ async function handleEffectieveMandatarissen() {
   const mandatarissen = await fetchEffectiveMandatarissenWithoutBesluit();
```

Comment:
I would update this name, because you don't only check if there are mandatarissen that are effective without besluit, but are effective without besluit and without notification.

Reply:
generalized it